### PR TITLE
update sdk document

### PR DESCRIPTION
**Pull request by JobSmithManipulation**
### What problem does this PR solve?

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

### Type of change

- [x] New Feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by JinHai-CN** [comment]
Would you please use English to write the code comment?

File: sdk/python/ragflow/ragflow.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def create_assistant(self, name: str = "assistant", avatar: str = "path", knowle
             datasets.append(dataset.to_json())
 
         if llm is None:
-            llm = Assistant.LLM(self, {"model_name": None,
+            llm = Assistant.LLM(self, {"model_name": "None",
```
Comment (KevinHuSh) [inline sdk/python/ragflow/ragflow.py]:
```suggestion
            llm = Assistant.LLM(self, {"model_name": None,
```

File: sdk/python/ragflow/ragflow.py [hunk 2]
```diff
@@ -30,8 +30,8 @@ def __init__(self, user_key, base_url, version='v1'):
         self.api_url = f"{base_url}/api/{version}"
         self.authorization_header = {"Authorization": "{} {}".format("Bearer", self.user_key)}
 
-    def post(self, path, param, stream=False):
-        res = requests.post(url=self.api_url + path, json=param, headers=self.authorization_header, stream=stream)
+    def post(self, path, param,stream=False):
```
Comment (KevinHuSh) [inline sdk/python/ragflow/ragflow.py]:
```suggestion
    def post(self, path, param, stream=False):
```

File: sdk/python/ragflow/ragflow.py [hunk 3]
```diff
@@ -31,7 +31,7 @@ def __init__(self, user_key, base_url, version='v1'):
         self.authorization_header = {"Authorization": "{} {}".format("Bearer", self.user_key)}
 
     def post(self, path, param, stream=False):
-        res = requests.post(url=self.api_url + path, json=param, headers=self.authorization_header, stream=stream)
+        res = requests.post(url=self.api_url + path, json=param, headers=self.authorization_header,stream=stream)
```
Comment (KevinHuSh) [inline sdk/python/ragflow/ragflow.py]:
```suggestion
        res = requests.post(url=self.api_url + path, json=param, headers=self.authorization_header, stream=stream)
```

File: sdk/python/ragflow/modules/base.py [hunk 4]
```diff
@@ -19,7 +19,7 @@ def to_json(self):
         return pr
 
     def post(self, path, param, stream=False):
-        res = self.rag.post(path, param, stream=stream)
+        res = self.rag.post(path, param,stream=stream)
```
Comment (KevinHuSh) [inline sdk/python/ragflow/modules/base.py]:
```suggestion
        res = self.rag.post(path, param, stream=stream)

```

File: sdk/python/ragflow/modules/base.py [hunk 5]
```diff
@@ -20,6 +20,7 @@ def to_json(self):
 
     def post(self, path, param, stream=False):
         res = self.rag.post(path, param, stream=stream)
+
```
Comment (KevinHuSh) [inline sdk/python/ragflow/modules/base.py]:
```suggestion
```

**Pull request merged**